### DISCUSSION TITLE
VP9 Packetizer Support

### DISCUSSION THREAD

**Discussion by danielkoohmarey-bc** [shinyoshiaki/rainy-webrtc#159] [Q&A]
I see you have a depacketizer implemented that parses the data buffer in https://github.com/shinyoshiaki/werift-webrtc/blob/53ed05e7be9e26b205f104e3a96b05f714b5d11d/packages/rtp/src/codec/vp9.ts, but do you support encoding/serializing a raw vp9 payload into a complete RTP packet by adding the payload description/header?

**Comment by shinyoshiaki** [comment]
I'd like to be able to packetize VP9 and other codecs at some point, but it's not a priority at the moment.

If you don't mind, can you tell me what you want to use it for?
